### ISSUE TITLE
Dev container: ssh-add: communication with agent failed

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode-remote-release#7601]
Testing #7587

Dev Containers v0.265.0
Version: 1.74.0-insider (user setup)
Commit: 8b9891739b703b50547f506a7e2bc9565e52beff

Following the steps from #7587, env variable is set, `ssh-add -l` returns entry when in local. but not in devcontainer

![image](https://user-images.githubusercontent.com/6461412/204538410-c78dec8e-5c25-40ab-a2a3-1e5c8bf470ca.png)

**Comment by joyceerhl** [comment]
I also see this even with the following devcontainer to ensure SSH is installed, so am blocked on testing for now.

```json
{
	"image": "mcr.microsoft.com/devcontainers/base:jammy",
	"features": {
		"ghcr.io/devcontainers/features/sshd:1": {}
	}
}
```
 
![image](https://user-images.githubusercontent.com/30305945/204622507-c81cc8f2-a36f-4878-8296-4f8fdd8c7da4.png)

**Comment by chrmarti** [comment]
Fixed in Dev Containers 0.266.0-pre-release. @aeschli @joyceerhl Could you retry with that? Thanks!
